ui/webui/test/check-storage

```diff
@@ -495,13 +496,13 @@ class TestStorageMountPoints(anacondalib.VirtInstallMachineCase):
         r.check_disk_row(disk, row, "Mount point", "/boot")
         r.check_disk_row_reformatted(disk, row, "Reformat")
         row = 3
-        r.check_disk_row(disk, row, "Partition", f"vda{row}")
```

Comment:
This changes a lot in https://github.com/rhinstaller/anaconda/pull/4975/files 

You will need to adjust